crates/viewer/re_selection_panel/src/item_heading_with_breadcrumbs.rs

```diff
@@ -0,0 +1,317 @@
+//! The heading of each item in the selection panel.
+//!
+//! It consists of a blue background (selected background color),
+//! and within it there are "bread-crumbs" that show the hierarchy of the item.
+//!
+//! A > B > C > D > item
+//!
+//! Each bread-crumb is just an icon or a letter.
+//! The item is an icon and a name.
+//! Each bread-crumb is clickable, as is the last item.
+//!
+//! The bread crumbs hierarchy should be identical to the hierarchy in the
+//! either the blueprint tree panel, or the streams/time panel.
+
+use re_chunk::EntityPath;
+use re_data_ui::item_ui::{cursor_interact_with_selectable, guess_instance_path_icon};
+use re_entity_db::InstancePath;
+use re_log_types::EntityPathPart;
+use re_ui::{icons, list_item, DesignTokens, SyntaxHighlighting, UiExt as _};
+use re_viewer_context::{Contents, Item, SpaceViewId, ViewerContext};
+use re_viewport_blueprint::ViewportBlueprint;
+
+use crate::ItemTitle;
+
+const ICON_SCALE: f32 = 0.5; // Because we save all icons as 2x
+
+/// We show this above each item section
+pub fn item_heading_with_breadcrumbs(
+    ctx: &ViewerContext<'_>,
+    viewport: &ViewportBlueprint,
+    ui: &mut egui::Ui,
+    item: &Item,
+) {
+    re_tracing::profile_function!();
+
+    ui.list_item()
+        .with_height(DesignTokens::title_bar_height())
+        .interactive(false)
+        .selected(true)
+        .show_flat(
+            ui,
+            list_item::CustomContent::new(|ui, context| {
+                ui.allocate_new_ui(
+                    egui::UiBuilder::new()
+                        .max_rect(context.rect)
+                        .layout(egui::Layout::left_to_right(egui::Align::Center)),
+                    |ui| {
+                        ui.spacing_mut().item_spacing.x = 4.0;
+
+                        {
+                            // No background rectangles, even for hovered items
+                            let visuals = ui.visuals_mut();
+                            visuals.widgets.active.bg_fill = egui::Color32::TRANSPARENT;
+                            visuals.widgets.active.weak_bg_fill = egui::Color32::TRANSPARENT;
+                            visuals.widgets.hovered.bg_fill = egui::Color32::TRANSPARENT;
+                            visuals.widgets.hovered.weak_bg_fill = egui::Color32::TRANSPARENT;
+                        }
+
+                        // First the C>R>U>M>B>S>
+                        {
+                            let previous_style = ui.style().clone();
+                            // Dimmer colors for breadcrumbs
+                            let visuals = ui.visuals_mut();
+                            visuals.widgets.inactive.fg_stroke.color = egui::hex_color!("#6A8CD0");
```

Comment:
would be great to have a TODO here with the hue/scale, so I can update it with the ColorToken pr